tests/test_benchmarker.py

```diff
@@ -326,6 +343,7 @@ def test_append_to_results(self, benchmark_result, results_path):
         "model has not been benchmarked",
         "model few-shot has not been benchmarked",
         "model few-shot has been benchmarked",
+        "model few-shot has been benchmarked, but not generative",
```

Comment:
Martin suggested (and I have had a few successes with it, but haven't tried it more than a few times) to create a test object when test-cases becomes quite large. E.g. `ModelHasBeenBenchmarkedCase`.